checkov/terraform/checks/resource/gcp/GoogleComputeDefaultServiceAccount.py

```diff
@@ -29,9 +29,11 @@ def scan_resource_conf(self, conf):
             self.evaluated_keys = ['name']
             return CheckResult.PASSED
         self.evaluated_keys = ['service_account/[0]/email', 'name']
-        if 'source_instance_template' in conf.keys() and 'service_account' not in conf.keys():
-            # the absence of service_account from a 'google_compute_instance_from_template' definition does not imply
-            # usage of the default service account.
+        if 'service_account' not in conf:
+            if self.entity_type == 'google_compute_instance_template' and 'source_instance_template' in conf:
```

Comment:
Why do we need this inner statement if we are going to return `UNKNOWN` anyways?

Reply:
Good call. Updated.